artifact-management/commands/gem-test-core.yml

```diff
@@ -85,6 +85,7 @@ steps:
         - run:
             name: Test process
             command: |-
+              set -e
```

Comment:
I think we should be using `set -e` everywhere where we have multi-line scripts, not just here.

Reply:
Yes. I am agree. I was reading if it was exactly what we want because some people do not agree with using set -e. 
http://mywiki.wooledge.org/BashFAQ/105
But I think is not our case.